src/Dialog/Dialog.js

```diff
@@ -10,6 +10,8 @@ import Paper from '../Paper';
 import ReactTransitionGroup from 'react-addons-transition-group';
 
 class TransitionItem extends Component {
```

Comment:
That was intentional, since `TransitionItem` and `DialogInline` don't get a separate export. Perhaps these components should be broken out into separate files.